src/main/webapp/app/exercises/programming/manage/services/programming-exercise-participation.service.ts

```diff
@@ -185,4 +185,10 @@ export class ProgrammingExerciseParticipationService implements IProgrammingExer
     retrieveCommitHistoryForTemplateSolutionOrTests(exerciseId: number, repositoryType: string): Observable<CommitInfo[]> {
         return this.http.get<CommitInfo[]>(`${this.resourceUrl}${exerciseId}/commit-history/${repositoryType}`);
     }
+
+    retrieveCommitHistoryForAuxiliaryRepository(exerciseId: number, repositoryType: string, auxiliaryRepositoryId: number): Observable<CommitInfo[]> {
+        const params: { [key: string]: number | string } = {};
```

Comment:
Would this work here?

```suggestion
        const params: { [key: string]: number } = {};
```